.azure/gpu-tests-pytorch.yml

```diff
@@ -55,6 +55,9 @@ jobs:
         "Lightning | latest":
           image: "pytorchlightning/pytorch_lightning:base-cuda-py3.11-torch2.3-cuda12.1.0"
           PACKAGE_NAME: "lightning"
+        "Lightning | future":
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.11-torch2.4-cuda12.1.0"
```

Comment:
Later you use `cu124` so shall we bump also the image to be aligned?

Reply:
I suggest not to use 12.4 now, because it's not working on our GPU CI cluster (I tried already). The machines probably need a driver update first, but this could be done later as a follow up.